### PR TITLE
test: refactor the cli stop test

### PR DESCRIPTION
**Pull request by ZYecho**
Signed-off-by: zhangyue <zy675793960@yeah.net>

### Ⅰ. Describe what this PR did
1. add  inspect filter func 
2. use 1 to refactor the test case in cli stop test
3. add some clean up operation

### Ⅱ. Does this pull request fix one issue?

None.

### Ⅲ. Why don't you add test cases (unit test/integration test)? (你真的觉得不需要加测试吗？)
None.

### Ⅳ. Describe how to verify it
`go test -gocheck.f PouchStopSuite`

### Ⅴ. Special notes for reviews
None.

### CODE REVIEW

File: test/utils.go [hunk 1]
```diff
@@ -95,3 +97,12 @@ func IsTLSExist() bool {
 	}
 	return true
 }
+
+func inspectFilter(name, filter string) (string, error) {
+	format := fmt.Sprintf("{{%s}}", filter)
+	result := command.PouchRun("inspect", "-f", format, name)
+	if result.Error != nil || result.ExitCode != 0 {
+		return "", fmt.Errorf("failed to inspect %s: %s", name, result.Combined())
```
Comment (allencloud) [inline test/utils.go]: could you update the error message to insert more info about filter? @ZYecho 
like `fmt.Errorf("failed to inspect %s via filter %s: %s", name, filter, result.Combined())`

File: test/utils.go [hunk 2]
```diff
@@ -95,3 +97,12 @@ func IsTLSExist() bool {
 	}
 	return true
 }
+
+func inspectFilter(name, filter string) (string, error) {
```
Comment (allencloud) [inline test/utils.go]: This encapsulated function looks so good to me. 😄 🌹 🍻

**Comment by allencloud** [comment]
This is really what we need in pouchcontainer's test. Thanks a lot for your contribution. @ZYecho

**Comment by allencloud** [comment]
I search all the test case file, and I found that there are more than 10 times to appear `output := command.PouchRun("inspect", name).Stdout()`. And I think there are some cases in which we could take advantages of your encapsulated function `inspectFilter`.
 
We could replace those codes with `inspectFilter`.
Maybe in next pull request. @ZYecho 🍻

File: test/utils.go (hunk 2 above)
Comment (ZYecho) [reply test/utils.go]: done.

File: test/utils.go (hunk 1 above)
Comment (ZYecho) [reply test/utils.go]: done.

**Comment by ZYecho** [comment]
@allencloud  Sometimes the cri test in CI may fail, could you help to check this?

**Comment by allencloud** [comment]
> Sometimes the cri test in CI may fail, could you help to check this?

It is an existing issue of PouchContainer. However it will hardly influence user's experience on PouchContainer. @ZYecho 
For what causes this, it is IO handling of container. And @fuweid is working on it with the newly pr https://github.com/alibaba/pouch/pull/2375 put forward .
we have already recorded the the flaky test in https://github.com/alibaba/pouch/issues/2290.

**Comment by allencloud** [comment]
LGTM. Thanks again. I am merging this.

**Pull request merged**